_includes/sections/instant-messenger.html

```diff
@@ -1,83 +1,269 @@
 <h1 id="im" class="anchor"><a href="#im"><i class="fas fa-link anchor-icon"></i></a> Encrypted Instant Messenger</h1>
 
+<div>
+    <p>We only recommend instant messenger programs or apps that support <a href="https://en.wikipedia.org/wiki/End-to-end_encryption">end-to-end encryption (E2EE)</a>. When E2EE is used, all transmissions (messages, voice, video, etc.) are encrypted <strong>before</strong> they are sent from your device. E2EE protects both the authenticity and confidentiality of the transmission as they pass through any part of the network (servers, etc.).</p>
+
+    <p>All the client programs/apps we chose are <a href="https://en.wikipedia.org/wiki/Free_and_open-source_software">free and open-source software</a> unless otherwise mentioned. This to ensure that the code can be independently verified by experts now and in the future.</p>
+
+    <p>We have described the three main types of messaging programs that exist: Centralized, Federated and Peer-to-Peer (P2P), with the advantages and disadvantages of each.</p>
+
+    <h2 id="centralized" class="anchor"><a href="#centralized"><i class="fas fa-link anchor-icon"></i></a> Centralized</h2>
+
+    <p>Centralized messengers are those where every participant is on the same server or network of servers controlled by the same organization.</p>
+
+    <h3>Advantages</h3>
+    <ul>
+        <li>Easier for developers to implement and push new features.</li>
```

Comment:
What I wonder is, is this an advantage for the user?
```suggestion
        <li>New features and changes can be implemented more quickly.</li>
```